src/core/reducers/addons.js

```diff
@@ -261,7 +255,7 @@ export const isAddonLoading = (state: AppState, slug: string): boolean => {
     return false;
   }
 
-  return Boolean(state.addons.loadingBySlug[slug.toLowerCase()]);
+  return Boolean(state.addons.loadingByIdInURL[slug]);
```

Comment:
Do you think this instance of `slug` should be renamed to `addonId`?

Reply:
yes, thanks